Makefile

```diff
@@ -6,4 +6,4 @@ test:
 	$(GO) test ./... -coverprofile=coverage.out ./...
 	$(GO) vet ./...
 	gofmt -l .
-	[ "`gofmt -l .`" = "" ]
```

Comment:
shouldn't -race be added to the `test` line?

Reply:
[Per Dev Ops WG call, no.](https://github.com/edgexfoundry/go-mod-registry/issues/35#issuecomment-592093762)

Reply:
Ok, then. :-)